xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxTradeServiceRaw.java

```diff
@@ -172,43 +172,49 @@ public FtxResponse<List<FtxOrderDto>> getFtxOpenOrders(String subaccount, String
     }
   }
 
-  public OpenOrders getOrderHistoryForSubaccount(String subaccount, TradeHistoryParams params)
-          throws IOException {
-    if (params instanceof TradeHistoryParamCurrencyPair) {
+  public OpenOrders getOrderHistoryForSubaccount(String subaccount, TradeHistoryParams params) throws IOException {
+    if (params instanceof TradeHistoryParamsAll) {
```

Comment:
`TradeHistoryParamsAll` implements way more parameters then this implementation supports

I think we should stick to checking individual interfaces like `TradeHistoryParamCurrencyPair`